src/bank.rs

```diff
@@ -912,9 +913,62 @@ impl Bank {
 
     /// Process an ordered list of entries.
     pub fn process_entries(&self, entries: &[Entry]) -> Result<()> {
+        self.par_process_entries(entries)
+    }
+
+    pub fn first_err(results: &[Result<()>]) -> Result<()> {
+        for r in results {
+            r.clone()?;
+        }
+        Ok(())
+    }
+    pub fn par_execute_entries(&self, entries: &[(&Entry, Vec<Result<()>>)]) -> Result<()> {
+        inc_new_counter_info!("bank-par_execute_entries-count", entries.len());
+        let results: Vec<Result<()>> = entries
+            .into_par_iter()
+            .map(|(e, locks)| {
+                let results = self.execute_and_commit_transactions(
+                    &e.transactions,
+                    locks.to_vec(),
+                    MAX_ENTRY_IDS,
```

Comment:
For the record, this should be [`MAX_ENTRY_IDS / 2` like the other callsite](https://github.com/solana-labs/solana/pull/1499/files#diff-7145777648f8c12caeb15b4e1755215cL718), we fixed this almost 1.5 years later #9423. ;)